.github/workflows/main_housingrepairsschedulingapi.yml

```diff
@@ -27,6 +26,13 @@ jobs:
         run: dotnet tool update --global dotnet-format
       - name: Run dotnet format
         run: dotnet format --check --verbosity diagnostic
+      - name: Setup Terraform
+        if: always()

```

Comment:
why do we need this?

Reply:
`if: always()` will run that step regardless of whether or not the previous step succeeded. See [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-not-requiring-successful-dependent-jobs).

So, in this case, if the dotnet format step failed for some reason, it will still run the next step which is to setup terraform and check its format.

Otherwise, the terraform setup steps would be dependent on the success of the previous dotnet format step.